web-ui/src/main/resources/catalog/views/default/less/gn_topics_default.less

```diff
@@ -10,6 +10,9 @@
       margin-bottom: 0;
     }
   }
+  #chips-card {
```

Comment:
I don't believe it is good practice to apply style to multiple elements based on ID. Maybe add a class and use it as a selector instead?